### DISCUSSION TITLE
Labels and languageCode

### DISCUSSION THREAD

**Discussion by huwgit** [vendurehq/vendure#1943] [Q&A] [answered]
I am trying to add a label to a custom field in vendure.config:

            {
                name: 'extendedDescription', 
                type: 'localeString',
                label: [
                  { languageCode: LanguageCode.en, value: 'Extended description' },
                ],
           
       
              },

This comes up with an error on npm run dev:

src/vendure-config.ts(73,35): error TS2304: Cannot find name 'LanguageCode'. Any ideas?

**Comment by michaelbromley** [comment, marked as answer]
Hi,
You'll need to import the `LanguageCode` enum:
```ts
import { LanguageCode } from '@vendure/core';
```

**Reply by huwgit** [reply]
Thanks that was it
